src/pull_request.ts

```diff
@@ -278,3 +282,25 @@ async function submitReview(
     );
   }
 }
+
+function shouldIgnorePullRequest(pull_request: { body?: string }) {
+  const ignorePhrases = [
+    "@presubmit ignore",
+    "@presubmit: ignore",
+    "@presubmit skip",
+    "@presubmit: skip",
+    "@presubmitai ignore",
+    "@presubmitai: ignore",
+    "@presubmitai skip",
+    "@presubmitai: skip",
+  ];
```

Comment:
The current array of ignore phrases is repetitive and could be generated programmatically to reduce duplication and improve maintainability. Consider using a more structured approach:
```typescript
const commands = ['ignore', 'skip'];
const prefixes = ['@presubmit', '@presubmitai'];
const separators = ['', ': '];

const ignorePhrases = prefixes.flatMap(prefix =>
  commands.flatMap(command =>
    separators.map(sep => `${prefix}${sep}${command}`)
  )
);
```
This makes it easier to add new commands or variations while reducing the chance of typos.